### PR TITLE
Readme and folder feature

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
In this PR,  I have refactored my code and removed most of the redundant code. 

### Files to check:
- organizer.jsx
- folder.jsx
- readme.md

###Files to ignore
- most of the files have modifications only. You can ignore those.

### CODE REVIEW

File: src/components/modal/noteModal.css [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+}
+
+.modal-container{
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background: var(--overlay);
+    z-index: 4000;
```
Comment (Developers-codz) [inline src/components/modal/noteModal.css:16]: why this much z-index ???

**Review by Developers-codz — APPROVED** [review]
Good work on code, found nothing to suggest, u can go for merge 💯

File: src/components/modal/noteModal.css (hunk 1 above)
Comment (ApurvaSawant11) [reply src/components/modal/noteModal.css:16]: I am using react-pro-sidebar which has a greater z-index. To overlap that I had to use larger z-index value

File: src/components/card/ArchiveCard.jsx [hunk 2]
```diff
@@ -65,12 +61,14 @@ const ArchiveCard = ({ note, user }) => {
...
-      {openModal.state && (
-        <EditModal
-          openModal={setOpenModal}
-          note={openModal.note}
+      {showModal && (
+        <NoteModal
           user={user}
-          noteType="archivedNotes"
+          noteState={note}
+          // noteContent={note.content}
```
Comment (nayyyhaa) [inline src/components/card/ArchiveCard.jsx]: can remove console.log from here.

File: src/pages/organizer/Organizer.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            </div>
+          </div>
+        )}
+      </div>
+      <div className="flex-row-center wrap content-start mt-2p5">
+        {foldersList.map((folder, index) => {
+          return (
+            <NavLink
+              key={index}
+              to={`/organizer/${folder.split(" ").join("-")}`}
```
Comment (nayyyhaa) [inline src/pages/organizer/Organizer.jsx:63]: woah! nice way of setting link from folder-name.

**Review by nayyyhaa — APPROVED** [review]
Nice work. Proper use of destructuring, and template strings has been used. Found nothing to suggest. You can merge the changes.

**Review by omkarpatke — APPROVED** [review]
Everything looks good to merge.

File: src/components/card/ArchiveCard.jsx (hunk 2 above)
Comment (ApurvaSawant11) [reply src/components/card/ArchiveCard.jsx]: Remove it. Thanks!

**Pull request merged**